src/client.js

```diff
@@ -530,13 +530,14 @@ const Client = IgeClass.extend({
 						.mount(ige);
 
 					// old comment => 'Create the UI scene'
-					this.uiScene = new IgeScene2d()
+					// never used
+					/* this.uiScene = new IgeScene2d()
```

Comment:
What is the evidence that we never use `this.uiScene`?

Reply:
Hmm, maybe the original idea was to mount `IgeUiEntity` instances on this scene, but it changed some time ago.